modules/kubernetes/aks-core/modules.tf

```diff
@@ -504,7 +504,7 @@ module "starboard" {
   cloud_provider                  = "azure"
   client_id                       = var.starboard_config.client_id
   resource_id                     = var.starboard_config.resource_id
-  volume_claim_storage_class_name = "managed-csi-zrs"
+  volume_claim_storage_class_name = var.starboard_config.volume_claim_storage_class_name
```

Comment:
Did the same for Prometheus. The sad part is that  it can't be part of the object since it needs a default value, hence it has to be a separate variable :-(